dev-packages/e2e-tests/test-applications/aws-serverless-esm/tests/basic.test.ts

```diff
@@ -54,6 +58,19 @@ test('AWS Serverless SDK sends events in ESM mode', async ({ request }) => {
     }),
   );
 
+  expect(transactionEvent.spans).toContainEqual(
```

Comment:
apparently our check `isWrappedByOtel` is not working, so we still emit the span. it's not really a problem as this will just be a child, but weird...